tsc/meetings/2020-09-29.md

```diff
@@ -0,0 +1,179 @@
+VDBs namespace. Will peruse this and see if any issues arise.
+
+VDB changes
+https://github.com/AcademySoftwareFoundation/openvdb/pull/818#discussion_r496433146
+
+Although minor, agreed to split out VDB specific changes to a different PR.
+
+Deprecated methods and odd implementations
+https://github.com/AcademySoftwareFoundation/openvdb/pull/818#discussion_r496434030
+https://github.com/AcademySoftwareFoundation/openvdb/pull/818#discussion_r496441704
+
+For backwards compatibility and internal reasons, some design decisions were
+made which now look odd in the open source version of AX. These still need to
+exist in the standalone repo but can be removed and re-worked for the merge into
+VDB. Agreed to action these.
+
+Executable terminology
+https://github.com/AcademySoftwareFoundation/openvdb/pull/818#discussion_r496439529
+
+Agreeded to rename to Binary and see what it looks like.
+
+Executable member interface and usage
+https://github.com/AcademySoftwareFoundation/openvdb/pull/818#discussion_r496444627
+https://github.com/AcademySoftwareFoundation/openvdb/pull/818#discussion_r496443272
+
+In the current design, the executables store all settings on a sub struct unique
+ptr. This produces a clean API with separation of the trivial execute calls
+from the member settings. However it means that they must be copied to make
+modifications if the exe is const (though this copying is cheap) and requires
+explicit copy constructors/assignment operators. Another option would be to have
+users provide these settings on each call to execute. This better ties in some
+settings to the grid data being executed, produces a trivially copyable object
+and means the executable can be const. Nick, will play around with these ideas.
+
+Combination of Compiler to Executable
+https://github.com/AcademySoftwareFoundation/openvdb/pull/818#discussion_r496446789
+
+The templated compile methods exist for internal reasons and can be removed. It
+may be better to introduce static creators on the executables which take a compiler
+vs the friend implementation. No clear advantage currently, but agreeded that
+the workflow between the code generators, compiler and exes could use some
+attention.
+
+ax::run() with multiple grids/attribute bindings
+
+Agreed that this particular signature could use some more explicit behavior.
+Agreed to error if points and volumes are provided. Custom way to bind attributes
+to grids should exist that don't rely on the grid name (i.e. what if the grid is
+const).
+
+Below are the main action points which block the initial merge:
+ - Integer precision changes
+ - Modulo implementation
+ - Deprecated code removal
+ - lerp(), fit(), signbit() changes
+ - AX Versioning changes
+
+5) Simd
+
+Ken, working on some more intrinsic functionality for VDB. Nick, has code he'd
+like to upstream which uses SIMD wrappers. TSC is open to including external
+SIMD intrinsic wrapper software in VDB.
+
```

Comment:
Mind has gone blank on if we discussed anything else here, let me know if I've missed anything at all